lib/src/__tests__/DatePickerTestingLib.test.tsx

```diff
@@ -96,4 +105,77 @@ describe('<DatePicker />', () => {
     fireEvent.click(screen.getByLabelText('Jan 1, 2018'));
     expect(onChangeMock).not.toHaveBeenCalled();
   });
+
+  describe('input validation', () => {
+    const locales = {
```

Comment:
I'd propose use jest.each for such cases. It outputs to the same tests but requires less code and significantly more readable

Reply:
Regarding the usage of `jest.each`, we will move the test to mocha, so I think that it will be simpler to rewrite the current ones to a similar pattern without be tightened to any custom Jest API. I have seen https://github.com/ryym/mocha-each.